cliquet/resource/__init__.py

```diff
@@ -528,6 +532,15 @@ def process_record(self, new, old=None):
         :returns: the processed record.
         :rtype: dict
         """
+        if old is None or self.model.modified_field not in old:
+            return new
```

Comment:
This makes me realize that a `super()` from the examples of `process_record` in documention